Update landing page

VijaySharma815 opened the pull request.

GataKamsky commented:
@VijaySharma815 my review comments:

- let's make all references to 'fetchd' consistent and spelt all in lowercase.
- 'Visit fetch.ai developer forums:'  needs the indent but not the bullet
- Tech stack image needs a design revamp and each component should be clickable. @StephenBrownFetch will provide the design and @5A11 can provide the urls/links for the component sections

VijaySharma815 commented:
> * let's make all references to 'fetchd' consistent and spelt all in lowercase.
> * 'Visit fetch.ai developer forums:'  needs the indent but not the bullet

Addressed the comment.

5A11 commented:
Couple of comments:

- The first image, in its current size is illegible. Needs to be larger.
- Slightly tone down the buttons' hover behaviour (they pop up too much when hovered over).
- Could the ACN have its own hover behaviour (i.e. pop out when hovered over)?

devjsc commented on pages/notes.mdx:
```diff
@@ -1,5 +1,11 @@
+### 30th October 2023
+
+ - Updated [Agentverse concepts ↗️](/concepts#agentverse) resources.
+
```
let's have this in descending order; so newest updates at top

FelixNicolaeBucsa replied:
Solved. Will edit all other PRs as well.
